src/screens/main/portfolio/index.js

```diff
@@ -67,26 +68,25 @@ class Portfolio extends Component {
   }
 
   balanceCalculations = async () => {
-    const { tokenSymbolString, tokenBalances } = await this.formatTokens(this.state.data);    
+    const { tokenSymbolString, tokenBalances } = await this.formatTokens(this.state.data);
     await this.props.fetchCoinData(tokenSymbolString);
     await this.props.calculateWalletBalance(tokenBalances, this.props.tokenConversions); //amount of tokens and symbol -> token balance, conversions -> matrix of prices
-    await this.setState({ 
-      apiRequestString: tokenSymbolString, 
+    await this.setState({
+      apiRequestString: tokenSymbolString,
       walletBalance: this.props.walletBalance,
       tokenPrices: this.props.tokenBalances,
       tokenAmounts: tokenBalances,
-
     });
     this.showTokens();
   }
   /**
-   * tokens are passed into the function where their symbols and addresses are parsed out and stored in an array, 
+   * tokens are passed into the function where their symbols and addresses are parsed out and stored in an array,
    * which is then passed to processAllTokenBalances. The return is the concatenated string of symbols in wallet, and the amount
    * of each token the user has for a given private key.
    */
   formatTokens = async (tokenList) => {
     let tokenObjectList = [];
-    let privateKey; 
+    let privateKey;
```

Comment:
never used.

Reply:
When I look at this branch in active branches the code is being used.

formatTokens = async (tokenList) => {
    let tokenObjectList = [];
    let privateKey;
    for (let i = 0; i < tokenList.length; i++) {
      let tokenObj = {};
      tokenObj.symbol = tokenList[i].symbol;
      tokenObj.contractAddress = tokenList[i].address;
      tokenObjectList.push(tokenObj);
    }
    privateKey =  this.state.currentWallet.privateKey;
    return { tokenSymbolString, tokenBalances } = await processAllTokenBalances(privateKey, tokenObjectList);
  }

Reply:
What i meant is let statement is redundant. Just use const below.

Reply:
@GregTheGreek fixed